### DISCUSSION TITLE
Incentives for Participating in OMI

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#32] [General]
@madjin brought this up in our OKR workshop and there's been a lot of discussion on Discord since. How do we get people involved in OMI and keep them involved? It's a lot of work to develop specifications, lead meetings, and build prototypes. Why are we doing this? Why should we expect others to contribute? Are there financial incentives or is it more about shaping what we as a group think is the future we want to live in?

Jin also expressed interest in having a meeting in discord to discuss this synchronously. I think we should do that, but let's kick it off here and schedule and ad-hoc meeting so we can get into the nitty-gritty for those who are interested.

I'll start with why I'm involved. I'm doing this work because I think we need to from a moral perspective but also a financial perspective. I'd like to ensure that we are creating a new view into the internet that feels humane, not some dystopian view of cyberspace depicted in sci-fi novels. I'm also doing this work, because I want to work on projects in this space that would benefit from a more connected, open ecosystem rather than siloed, closed ones. I expect that is very similar to a lot of people in this group. I won't receive direct monetary gain from participating in these specifications, but the projects I'd like to work on can exist if we write them.

The larger players in this space are trying to establish their closed ecosystems where they set the rules and can push competitors out. In order for all our smaller projects to succeed, we need to work together. The network effects that we will receive from interoperability will lift us all up. I've already seen this with glTF. If your project can import/export glTF then you now have access to a massive content library. We need to continue creating interoperability points for identity and social systems so that the friction of joining our worlds and finding our friends goes down.

As to the the financial cost of participating in OMI, I get that these things take time. Each member will need to determine how much time they can realistically allocate to participating in our work. However, in many standards groups, you pay for membership. This is also a political group, we need to recognize that businesses are working together towards specifications, but we may also be competing for each-other's business. Co-operatition is a good thing! In our process, we will make many small decisions that could have huge effects on participants projects. Participating in OMI is exchanging time for influence. I mean all of this in the least cut-throat way possible, I think we're in this together for a set of shared goals, but if you need incentives, they are definitely already there.

**Comment by madjin** [comment]
Thanks a bunch for getting this thread started! I've written about my personal motivations and why I'm involved here in this blog post if interested: https://hackmd.io/@XR/motivation

[M3](https://github.com/m3-org/charter) had very similar goals and practices as OMI. As a founding member I helped to organize dozens of events with lightning talks (VODs: https://www.youtube.com/c/Exokit/videos), cross-platform field trips, and take notes for everything.

My experience of running M3 while also participating in web3 projects and DAOs has given me a unique perspective on how to structure mission driven communities to be more effective. This will be the subject of a future blog post, for now I'll share a few ideas in regards to this discussion:

**How do we get people involved in OMI and keep them involved?**

Having skin in the game is essential. As you mentioned even standards groups require membership dues. Many of the groups participating in w3c are companies which by extension are all value aligned themselves.

In M3 there was no shortage of amazing ideas that emerged from our discussions, what we needed is more coordination on execution. Shared ownership as a brand or organization brings about a sense of responsibility in fulfilling whatever role the member can do.

**Treasury**

Membership dues can go into a treasury for the organization. Having a treasury plays a role in coordinating the community. People can buy other members time in order to collaborate on a needed feature or more broadly for the promotion of our ideas.

**Recommendations**

- Keep track of the things people do
  - Can setup a discord channel (#did-a-thing)
- Decide on membership fees
  - Per month, season, annual, or lifetime
  - Anything is better than $0
- Outline paths to earn your way in
  - Note taking during meetings
  - Setting up a website
  - Designing swag / flyers
  - Contributing towards an OKR

**Crypto**

Blockchains are full of coordination games centered around shared data layers and open source software. Here's some recommended actions for easy ways for us to participate.

- Invite the NFT discord bot
  - We can mint/send tokens for free without needing to setup wallets
  - Tokens can represent achievements, tickets, or swag
  - It's been purpose built for organizations like OMI
- Matrix bridge Discord to Webaverse / Metafactory
  - Comms with communities implementing similar things
- Consider setting up a [Gitcoin](https://gitcoin.co/) page
  - Gitcoin Grants like crypto-style patreon, overall ~20M in results
  - Members can receive tips, projects can receive donations
  - Web3 / NFT projects are deeply invested in metaverse interop
    - Community that will champion us in public

**Reply by gobengo** [reply]
What do you think of https://sourcecred.io/ ?

**Reply by madjin** [reply]
I think its an amazing tool, glad you mentioned it! (Intro vid for anyone curious: https://www.youtube.com/watch?v=3ZBDiNvddG4)

I could get one of the authors to do a lightning talk on it sometime.
